lib/mediawiki_api/wikidata/wikidata_client.rb

```diff
@@ -47,6 +47,22 @@ def create_claim(entity_id, snaktype = "value", property_id, value_data)
         action(:wbcreateclaim, token_type: "csrf", entity: entity_id, snaktype: snaktype, property: property_id, value: value_data)
       end
 
+      def get_entities(ids)
+        action(:wbgetentities, token_type: false, ids: ids)
+      end
+
+      def get_entities_by_titles(titles, sites)
+        action(:wbgetentities, token_type: false, titles: titles, sites: sites)
+      end
+
+      def set_qualifier(claim_id, snaktype = "value", property_id, value_data)
```

Comment:
Should it be documented here what the `value_data` attribute looks like?

For example, for setting platform to Windows it'd be `{ "entity-type": "item", "numeric-id": 1406, "id": "Q1406" }`.

It'd be nice to have this merged, I'm using it for a script atm :)

Reply:
IMO this is good enough as this isn't a high level abstraction of the API anyway.